lib/l10n.js

```diff
@@ -617,18 +788,28 @@ var l10n = {
     },
     msg_election_day_tomorrow: {
         en: 'Election Day is tomorrow!',
+        es: 'Dia para votar es mañana! ',
+        ko: "내일이 선거 날입니다!"
     },
     msg_election_day_today: {
         en: 'Today is Election Day!',
+        es: 'Hoy es dia de votar!',
+        ko: "오늘이 선거 날입니다!"
     },
     msg_polling_place: {
         en: 'Your polling place is at {{location}} in {{location_city}}. {{results.polling_place.link}}',
+        es: 'Su lugar de votar esta {{ubicación}} {{ubicación_cuidad}} {{results.polling_place.link}}',
```

Comment:
change to `{{location}} {{location_city}}`

Reply:
updating